src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -1,44 +1,35 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToMany;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Objects;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
 
+@Setter
+@Getter
+@NoArgsConstructor
+@Entity
+@ToString
 public class Movie implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String title;
+    @ManyToMany
     private List<Actor> actors;
```

Comment:
```suggestion
    @ManyToMany
    @JoinTable() // todo: Fill properties "name", "joinColumns" and "inverseJoinColumns"
    private List<Actor> actors;
```
Example of @ManyToMany relationship from the course
https://mate.academy/learn/java-hibernate/java-entity-relations-and-life-cycle?learnItemsFilter=All&section=video&videoId=1049